src/Microsoft.PowerShell.Commands.Utility/commands/utility/Var.cs

```diff
@@ -693,6 +693,13 @@ public SwitchParameter PassThru
 
         private bool _passThru;
 
+        /// <summary>
+        /// Gets whether we will append to the variable if it exists.
+        /// </summary>
+        [Parameter]
+        [Experimental("PSRedirectToVariable", ExperimentAction.Show)]
```

Comment:
```suggestion
        [Experimental(ExperimentalFeature.PSRedirectToVariable, ExperimentAction.Show)]
```

You already have the const string defined in this class